config/user-commands.m4

```diff
@@ -26,10 +26,12 @@ AC_DEFUN([ZFS_AC_CONFIG_USER_COMMANDS_COMMON], [
 	AC_PATH_TOOL(DU, du, "")
 	AC_PATH_TOOL(ECHO, echo, "")
 	AC_PATH_TOOL(EGREP, egrep, "")
+	AC_PATH_TOOL(FALSE, false, "")
 	AC_PATH_TOOL(FDISK, fdisk, "")
 	AC_PATH_TOOL(FGREP, fgrep, "")
 	AC_PATH_TOOL(FILE, file, "")
 	AC_PATH_TOOL(FIND, find, "")
+	AC_PATH_TOOL(FIO, fio, "/usr/bin/fio")
```

Comment:
I'd also suggest adding `Requires: fio` to the `%package test` section of the spec file, `zfs.spec.in`.  This way when the zfs-test package get's installed it will automatically pull in the fio package as a dependency.